src/Cli/Microsoft.DotNet.Cli.Sln.Internal/Microsoft.DotNet.Cli.Sln.Internal.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyName>Microsoft.DotNet.Cli.Sln.Internal</AssemblyName>
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
     <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PublicSign Condition=" '$([MSBuild]::IsOSPlatform('Windows'))' == 'false' ">true</PublicSign>
```

Comment:
```suggestion
    <PublicSign Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'false' ">true</PublicSign>
```

OR

```suggestion
    <PublicSign Condition="!$([MSBuild]::IsOSPlatform('Windows'))">true</PublicSign>
```

Reply:
Indeed, thanks! Went with the former since the latter is hard to read (see earlier comments).